### PR TITLE
(docs) indexer readme

### PR DESCRIPTION
**Pull request by ponderingdemocritus**
minor PR on indexer docs and Dockerfile

### CODE REVIEW

File: crates/dojo-indexer/README.md [hunk 1]
```diff
@@ -30,13 +34,21 @@ Dojo Indexer is a command line tool that allows you to index data from a given D
...
+
+```
+export DOJO_WORLD=
+export APIBARA_NODE=
+export RPC_NODE=
+```
 
 ```
-cargo run --bin dojo-indexer <world> <node> <rpc> <database_url>
+cargo run --bin dojo-indexer DOJO_WORLD APIBARA_NODE RPC_NODE DATABASE_URL
```
Comment (tarrencev) [inline crates/dojo-indexer/README.md:48]: Hm I think we should avoid suggesting using ENV vars the passing them as args for now. In the future we could update the cli to read from the ENV directly and properly namespace them

**Review by tarrencev — COMMENTED** [review]
thanks for the contribution. i would prefer to revert the env var stuff for now. otherwise lgtm

File: crates/dojo-indexer/README.md (hunk 1 above)
Comment (fracek) [reply crates/dojo-indexer/README.md:48]: Clap has a [neat feature (with the `env` feature)](https://docs.rs/clap/latest/clap/_derive/#arg-attributes) where it accepts parameters both as arguments/flags and as env variables.

File: .gitmodules [hunk 2]
```diff
@@ -1,4 +0,0 @@
-[submodule "cairo"]
```
Comment (tarrencev) [inline .gitmodules:1]: this is still necessary unfortunately, since `cargo test` uses it right now

**Pull request merged**